Project-18-Food-order-app/src/hooks/useInput.js

```diff
@@ -0,0 +1,28 @@
+import { useState } from "react";
+
+function useInput(defaultValue, validationFn) {
+  const [enteredValues, setEnteredValues] = useState(defaultValue);
+
+  const [didEdit, setDidEdit] = useState(false);
+
+  const isValueValid = validationFn(enteredValues);
+
+  const handleInputChange = (event) => {
+    setEnteredValues(event.target.value);
+
+    setDidEdit(false);
```

Comment:
@sknepali Here setDidEdit is set to false on every input change. This might not be necessary